views/js/pciCreator/dev/audioRecordingInteraction/runtime/js/recorder.js

```diff
@@ -181,11 +174,17 @@ define([
             init: function init() {
                 var self = this;
 
+                provider = (config.isCompressed)
+                    ? mediaRecorderProvider(config)
+                    : webAudioProvider(config, assetManager);
+
+                this.initAudioContext();
+
                 return navigator.mediaDevices.getUserMedia({ audio: true })
                     .then(function(stream) {
-                        provider = (config.isCompressed)
-                            ? mediaRecorderProvider(config)
-                            : webAudioProvider(config, assetManager);
+                        if (audioContext && (audioContext.state !== 'running')) {
+                            audioContext.resume();
```

Comment:
It's also returning a Promise and the error is not handled (see my other comment).

Reply:
fixed.